docs/AT-Pop.md

```diff
@@ -0,0 +1,79 @@
+# Microsoft Graph PowerShell SDK: Access Token Proof of Possession (AT PoP) Capability
+
+## Overview
+
+This README provides comprehensive details on the Access Token Proof of Possession (AT PoP) functionality introduced in the Microsoft Graph PowerShell SDK. This feature enhances security by binding tokens to specific HTTP methods and URIs, ensuring they are used only for their intended purposes.
+
+## Table of Contents
+
+- [Key Features](#key-features)
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Usage Examples](#usage-examples)
+- [References](#references)
+
+## Key Features
+
+- **Access Token Proof of Possession (AT PoP)**: This feature binds tokens to specific HTTP methods and URIs, preventing misuse of tokens by ensuring they are used only for the intended HTTP requests.
+- **Updated Dependencies**: Compatibility improvements with recent library changes.
+- **Enhanced Token Acquisition Options**: Users can now specify the HTTP method and URI during token acquisition to further secure token usage.
+
+### Token acquisition behaviors
+
+| Condition | Unbound (default) | Bound (PoP) |
+|-----------|-----------|-----------|
+| First sign-in | New token, interactive| New token, interactive |
+| Existing token, same URI | No new token, silent | No new token, silent |
+| Existing token, different URI | No new token, silent | New token, silent |
+| Existing expired token, below max token refreshes | New token, silent | New token, silent |
+| Existing expired token, exceeded max refreshes | New token, interactive | New token, interactive |
+
+## Installation
+
+To install the Microsoft Graph PowerShell SDK with the latest updates, use the following command:
+
+```powershell
+Install-Module -Name Microsoft.Graph -AllowClobber -Force
```

Comment:
We can leave out this section -- installation of this module is not changed by this feature.